Umweltzone/src/main/res/values-sv/ckchangelog.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?><!--
+  Copyright (C) 2019  Tobias Preuss
+
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+<resources xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
+    <!--
+        Overwrites ckChangeLog internal settings for all languages.
+        Will be removed when this application will be localized.
+    -->
+    <string name="changelog_title">Detta är nytt</string>
+    <string name="changelog_show_full">Mer &#8230;</string>
+
+    <string name="changelog_version_format">
+        Version <xliff:g id="version_name">%s</xliff:g>
+    </string>
+</resources>
```

Comment:
You are missing the translation for `changelog_error_resource_not_android_5`. See _values/ckchangelog.xml_.